src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -66,9 +78,9 @@ public JsonAdaptedPerson(Person source) {
      */
     public Person toModelType() throws IllegalValueException {
         final List<Tag> personTags = new ArrayList<>();
```

Comment:
What does this array list of tags do?

Reply:
Nothing this is original code. I didn't delete it cos it will cause a lot of problems with test cases

Reply:
Which test does it fail? is it possible to edit the test instead of just leave that hanging?

Reply:
Cos since the constructor of Person change right, I have to edit every single test case that uses person and not just the one related to my part. And its like every single class inside the test folder. Honestly I would change if it wasnt for codecov.  Initially i tried updating my test cases. But then codecov doesnt accept that other test cases fail.

Reply:
hmm like that then we delete first. I take it that its currently a headache to edit right now, with all the incoming changes